src/schema/types.ts

```diff
@@ -147,3 +148,27 @@ export type DataTrackedAttrs = {
   userID: string
   createdAt: number
 }
+
+export enum SectionGroup {
```

Comment:
It might be simpler to just use a union type:

`export type SectionGroup = 'body' | 'backmatter'`

Reply:
Done.